### DISCUSSION TITLE
User highlights are prioritized over message highlights

### DISCUSSION THREAD

**Discussion by anqueue** [Chatterino/chatterino2#3871] [Ideas] [answered]
In the highlights settings page it says that "Message highlights are prioritized over badge highlights, but under user highlights". While this is accurate on how Chatterino handles highlights, I don't think it is how it should be. Pre #3399 message highlights were above user highlights (even though it was stated otherwise in the UI) which is how I'm sure many people were used to.

A message highlight can be much more specific than a user highlight. I have a user highlight on someone and they ping me but the user highlight is prioritized, making it possible for me to miss that message (i have "@AuroR6S" as a message highlight):
![image](https://user-images.githubusercontent.com/35087590/180074054-6dfcd513-d246-457e-ae77-926e538d20f8.png)

**Comment by leon-richardt** [comment, marked as answer]
Related: https://github.com/Chatterino/chatterino2/discussions/3865
